ServiceInnovationLab/pancake-backend
# Feature/councils

# What has changed and why?

closes https://github.com/ServiceInnovationLab/pancake-frontend/issues/451
closes https://github.com/ServiceInnovationLab/pancake-frontend/issues/438
closes https://github.com/ServiceInnovationLab/pancake-frontend/issues/449

# How to test this change

1. run data base migration 
1. log in 
1. Check Tauranga exists
1. Make new councils
1. Delete councils
1. Enable and disable councils
1. Check you can't delete councils that have rebate forms recorded
1. Go to api docs - check the api/v1/councils method returns active councils
